docs/Proposing-a-new-module.md

```diff
@@ -0,0 +1,40 @@
+[Back to overview](./README.md)
```

Comment:
@bethanylang Can you add an entry for this also to the overview / TOC? I'd suggest right above the "Writing a module" entry.

Reply:
@felixarntz Done! https://github.com/WordPress/performance/pull/337

Reply:
@bethanylang Thank you! I think technically (or for the future) it would have been simpler to add it to this PR here, since now the other PR depends on the one here to be merged first. But this works :)

Reply:
@felixarntz Ah of course, will keep that in mind for the future.